Taskfile.yml

```diff
@@ -41,7 +41,7 @@ tasks:
     desc: Run tests
     dir: action
     cmds:
-      - go test {{.CLI_ARGS}} -covermode=count -coverprofile coverage.out ./...
+      - go test {{.CLI_ARGS}} -timeout 0 -shuffle=on -covermode=count -coverprofile coverage.out ./...
```

Comment:
Do we implement a timeout in the CI at the moment? By completely disabling the timeout you could potentially create a CI job that'll run for 24h without yielding a result in case of a faulty test. I'd rather prefer setting the timeout really high instead of disabling it completely, e.g. one hour or so

Reply:
Go has default timeout of 10 minutes ... found out the hard way.

Reply:
Also, it is kinda hard to gauge what timeout to set since on my PC it takes around 5 minutes to run these tests, but in CI it takes almost 20 minutes.

Reply:
I know, CI might take longer and 10min are already really short given for the complexity and time those tests take. That's why I proposed setting the timeout really high to be on the safe side but also not risk a never ending test to hog the CI. Maybe let's do 60min for now?

Reply:
OK, can do.